tests/test_java_to_minimal_controllers.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+cd /home/saluser/repos/ts_sal/bin
+passed=0
+failed=0
+
+echo "############# Begin Java Tests #############"
+echo "******** Test Java to Java ********"
+echo "Starting Java Commander"
+./minimal_java_commander.sh 1 23 > java_commander.log 2>&1 &
+sleep 5
+
+echo "Starting Java Controller"
+./minimal_java_controller.sh 1 23 > java_controller.log 2>&1
+JAVA_CONTROLLER=$( cat java_controller.log 2>&1 )
+JAVA_COMMANDER=$( cat java_commander.log )
+
+echo "======== Java to Java Test Results ========"
```

Comment:
Same comment as above. In fact this could be exactly the same function in both cases -- stored in a separate bash script.

Reply:
I think I got it.